src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -48,6 +49,7 @@
   private TsoRequest tsoReq;
   private volatile LeaderWrapper leaderWrapper;
   private ScheduledExecutorService service;
+  private IsolationLevel isolationLevel;
```

Comment:
Why put it here? Isolation level is neither a party of PDClient nor a part of region info.